packages/ui/src/components/va-rating/components/VaRatingItem/VaRatingItem.vue

```diff
@@ -101,14 +104,19 @@ export default defineComponent({
 
     watch(hoveredValue, () => emit('hover', hoveredValue.value || RatingValue.EMPTY))
 
+    const { keyboardFocusClass, keyboardFocusListeners } = useKeyboardFocusClass(props, 'va-rating-item')
```

Comment:
const componentName = kebabCase(getCurrentInstance()?.name)

Reply:
I've added this option, but it's not universal - some components should pass not root class.